ci/test.sh

```diff
@@ -212,3 +238,4 @@ case $provider in
     convox apps delete ci1 --wait
     ;;
 esac
+convox rack params set Internal=No
```

Comment:
Can you add this line at the end of the test? after `convox exec` https://github.com/convox/rack/blob/e57f080af66d797c9644c77604ee1d7743432bb8/ci/test.sh#L170